onnx/defs/math/defs.cc

```diff
@@ -3472,6 +3472,9 @@ ONNX_OPERATOR_SET_SCHEMA(
           }
 
           auto& input_shape = getInputShape(ctx, 0);
+          if (input_shape.dim_size() < 2) {
+            fail_shape_inference("First input should have at least 2 dimensions in ", ctx.getDisplayName(), ".");
```

Comment:
Would it be helpful to print the actual dim? That’s usually what helps diagnose issues better

Reply:
We should review all the error messages. Let's keep that for another PR.